src/components/TrainingEventCard.tsx

```diff
@@ -0,0 +1,117 @@
+import { TrainingSupbaseDataType } from '@data/types/trainingTypes';
+import { formatDate } from '@utils/index';
+import React from 'react';
+import styled from 'styled-components';
+import { priceFormatter } from './ProductCard';
+import CTA, { CTALink } from './CTA';
+
+interface TrainingEventCardProp {
+	training: TrainingSupbaseDataType;
+}
+
+const TrainingEventCard: React.FC<TrainingEventCardProp> = ({ training }) => {
+	return (
+		<TrainingEventWrapper>
+			<div>{training.title}</div>
+			<div>{`${formatDate(new Date(training.startDateTime))}`}</div>
+			<div>{`${formatDate(new Date(training.endDateTime))}`}</div>
+			<div>{training.trainingFormat}</div>
+			<div>{priceFormatter.format(training.price)}</div>
+			<CTAGroup>
+				<CTALink
+					className="no-animate training"
+					href={`/admin/temp/add-trainings/${training.id}`}>
+					view
+				</CTALink>
+				<CTA className="training delete">Delete</CTA>
+			</CTAGroup>
+		</TrainingEventWrapper>
+	);
+};
+
+export default TrainingEventCard;
+
+// export const getServerSideProps = withPageAuthRequired({
```

Comment:
getserversideprops shouldn't be in a component

Reply:
Ok. deleted